packages/frontend/src/pages/CreateProject.tsx

```diff
@@ -24,6 +24,16 @@ const CreateProject: FC = () => {
         );
     }
 
+    if (health.status === 'success' && !health.data?.isAuthenticated) {
```

Comment:
How come we need this in addition to `PrivateRoute` ?

Reply:
This route is outside the <PrivateRoute>. I can move this route inside it and change slightly the PrivateRoute logic to handle this since it would redirect to the welcome page instead the logic page.